src/router/resolvers/Resolver.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import { Component } from "vue-property-decorator";
+import { Route } from "vue-router";
+
+// route resolvers
+import {
+    InvokeResolver
+  } from "./index";
+
+Component.registerHooks(["beforeRouteEnter"]);
+@Component({
+    template:'<div></div>'

```

Comment:
is this necessary with the `<template><div></div></template>` above? (shrugs)

Reply:
Yeah, unfortunately, Vue complains about an empty template